go.mod

```diff
@@ -12,3 +12,5 @@ require (
 	github.com/stretchr/testify v1.3.0 // indirect
 	gopkg.in/inf.v0 v0.9.1
 )
+
```

Comment:
That's good can we have that in a separate commit.

Reply:
reverted change on go.mod & go.sum